aws/logs_monitoring/lambda_function.py

```diff
@@ -572,6 +565,25 @@ def s3_handler(event, context, metadata):
     bucket = event["Records"][0]["s3"]["bucket"]["name"]
     key = urllib.unquote_plus(event["Records"][0]["s3"]["object"]["key"]).decode("utf8")
 
+    # Common attributes for the events we find
+    aws_attributes = {
+        "aws": {
+            "s3": {
+                "bucket": bucket, 
+                "key": key
+            }
+        }
+    }
+
+    # Build a structured message out of the s3 event data.
+    def reformat_record(record):
```

Comment:
nits
```suggestion
    def build_message(record):
```
or
```suggestion
    def to_message(record):
```

Reply:
No problem.  Renamed to `build_message`.